index.html

```diff
@@ -1050,12 +1050,12 @@
 <div class="practice">
 
 <p><span class="practicelab" id="user-agent-data-sharing">
-User agents should only share and present information about the user that
-the user can reasonably anticipate being shared, and which directly relate to the users overt, immediate goals.
+User agents should only share information about the user that the user can reasonably
+anticipate being shared, and which directly relate to the user's goals.</span></p>
```

Comment:
Another simpler change here might be 
```suggestion
anticipate being shared, and which are needed for the user's goals.</span></p>
```
I've disliked "relate to" elsewhere for being vague, and I was overruled in those cases, but maybe this one will stick.